uvicorn/config.py

```diff
@@ -112,6 +112,20 @@ def create_ssl_context(certfile, keyfile, ssl_version, cert_reqs, ca_certs, ciph
     return ctx
 
 
+def ipvXmessage(version=None):
```

Comment:
I have some concerns about the function.

First of all, it's a kind of a public function (at least the name doesn't start with `_`) and from my point of view it means that user can (technically) call the function. And from my point of view the naming is not pythonic, I'd change the signature to 

```suggestion
def _get_server_start_message(host_ip_version: int = 4) -> Tuple[str, str]:
```